app/src/main/java/com/amaze/filemanager/utils/files/FileUtils.java

```diff
@@ -662,17 +682,19 @@ public static boolean canGoBack(Context context, HybridFile currentFile) {
         }
     }
 
-    public static long[] getSpaces(HybridFile hFile, Context context, final OnProgressUpdate<Long[]> updateState) {
+    public static Long[] getSpaces(HybridFile hFile, Context context, final OnProgressUpdate<Long[]> updateState) {
```

Comment:
Why `long` -> `Long`?

Reply:
in com/amaze/filemanager/asynchronous/asynctasks/LoadFolderSpaceDataTask.java:
    protected Pair<String, List<PieEntry>> doInBackground(Void... params) {
        **Long[] dataArray = FileUtils.getSpaces(file, context, this::publishProgress);**

        if (dataArray[0] != -1 && dataArray[0] != 0) {
            long totalSpace = dataArray[0];

          List<PieEntry> entries = createEntriesFromArray(dataArray, false);



As I remember,  at first I was useing long[], but there would be an error. So I make the change.

Reply:
Please don't use objects instead of primitives, it makes expensive autoboxing calls.

Reply:
At first, I tried to use primitives, but there'are some errors. I'm afraid there would be more if I try to change them. Also I try the performance of Long[], it seems to be ok.

Reply:
yes